index.d.ts

```diff
@@ -661,6 +659,10 @@ interface DurableObject {
   fetch: (request: Request, init?: RequestInit) => Promise<Response>;
 }
 
+interface DurableObjectStub {
```

Comment:
Is it intentional to have both DurableObject and DurableObjectStub?

Reply:
Yes, because it distinguishes between the DurableObject type that your class would implement vs the stub that would be returned from `namespace.get()`

Reply:
If we're punting on the constructor conversation from above, I'd punt on this interface for DurableObject as well... 

If you really want to keep them both, I'd add docstrings distinguishing them and include the name and id properties that the stub has, just so they're not confusingly identical

Reply:
I'll add the docstring and those properties

Reply:
May want to add documentation for those stub properties here
https://developers.cloudflare.com/workers/runtime-apis/durable-objects#object-stubs